modules/gatk4/leftalignandtrimvariants/main.nf

```diff
@@ -0,0 +1,46 @@
+process GATK4_LEFTALIGNANDTRIMVARIANTS {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::gatk4=4.2.6.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gatk4:4.2.6.1--hdfd78af_0':
+        'quay.io/biocontainers/gatk4:4.2.6.1--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(vcf), path(tbi)
```

Comment:
Maybe add the option for intervals here too

Reply:
https://gatk.broadinstitute.org/hc/en-us/articles/360037225872-LeftAlignAndTrimVariants#--intervals